test_prebuild/test_unit_conv/CMakeLists.txt

```diff
@@ -0,0 +1,98 @@
+#------------------------------------------------------------------------------
+cmake_minimum_required(VERSION 3.10)
```

Comment:
I know this is a straightforward merge but perhaps this should be updated sometime?

Reply:
Certainly. I can go through all files after this update and bump the minimum version to what is in the top-level CMakeLists.txt.